action.yaml

```diff
@@ -80,24 +80,24 @@ inputs:
     required: false
 
   # Used to construct the binary download URL
-  flakehub-push-binary:
```

Comment:
Unfortunately we'll need to keep the old yaml options in here, but we don't have to document them in the readme. Maybe add (deprecated) at the end of the description.

I did a search of GitHub repos, and there are a few out there I saw using `flakehub-push-branch`.

Reply:
Ah right, forgot that those need to be declared in `action.yaml` to be valid inputs